ceph-installer-rpm/build/build

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+sudo yum -y install epel-release
+sudo yum -y install fedpkg
+
+# Mock configuration file.
+# TODO: Use the CentOS Storage SIG repository instead of this Fedora Copr repo.
+# (discussion ongoing on the centos-devel list)
+MOCK_CFG=https://fedorapeople.org/~ktdreyer/ceph-installer/ktdreyer-ceph-installer.cfg
```

Comment:
I've added it into ceph-installer itself at https://github.com/ceph/ceph-installer/pull/172